xl-op/digitalai/dai-release-remote-runner/values-cli.yaml.tmpl

```diff
@@ -1,10 +1,11 @@
 replicaCount: {{ .RemoteRunnerCount }}
 
-{{- if eq .IsRemoteRunnerTruststoreEnabled true }}
+  {{- if eq .IsRemoteRunnerTruststoreEnabled true }}
 runner:
-  truststore: {{ .RemoteRunnerTruststore }}
-  truststorePassword: {{ .RemoteRunnerTruststorePassword }}
-{{- end }}
+  truststore: |-
+  {{ .RemoteRunnerTruststore | indent 4 }}
```

Comment:
indentation of 2 is already here in this line. Should this indentation be removed since there is indent 4. i.e. this line should be at same level as runner.